<ui version="4.0" >
 <class>ClientsDialog</class>
 <widget class="QDialog" name="ClientsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Clients</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons/resources.qrc" >:/32/clients.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Find:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="findlineEdit" />
       </item>
       <item>
        <widget class="QToolButton" name="cleartoolButton" >
         <property name="text" >
          <string>x</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView" >
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addpushButton" >
         <property name="text" >
          <string>Add</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelpushButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findlineEdit</tabstop>
  <tabstop>cleartoolButton</tabstop>
  <tabstop>addpushButton</tabstop>
  <tabstop>cancelpushButton</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cleartoolButton</sender>
   <signal>clicked()</signal>
   <receiver>findlineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>253</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel" >
     <x>230</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelpushButton</sender>
   <signal>clicked()</signal>
   <receiver>ClientsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>226</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel" >
     <x>269</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
